JPF/bad_buildings.sql

```diff
@@ -477,26 +643,8 @@ alter table pluto_nyc drop index bbl;
 alter table tb_changes_summary drop index ucbbl;
 alter table hpd_aep_list drop index full_addr;
```

Comment:
Are these drop index statements really necessary? If SQL errors out if attempting to add an index on something already indexed, I guess it is in order for this SQL file to be re-runnable. Maybe it's worth extracting the initial table cleaning to a separate SQL file? It would also shorten this file a little and make it easier to take in.